Cargo.toml

```diff
@@ -23,7 +23,7 @@ regex = "1"
 termcolor = "1"
 # Data (de)serialisation
 serde = { version = "1", features = ["derive", "rc"] }
-serde_json = "1"
+serde_json = { version = "1", features = ["preserve_order"] }
```

Comment:
I don't think this is being used any more?

Reply:
Probably not but I'll remove it and run the tests just in case.

Reply:
@lpil removed, cargo tests still pass.